core/templates/components/forms/custom-forms-directives/thumbnail-uploader.directive.ts

```diff
@@ -88,6 +89,7 @@ angular.module('oppia').directive('thumbnailUploader', [
               uploadedImage = null;
             }
             $scope.thumbnailIsLoading = false;
+            $scope.$applyAsync();
```

Comment:
This block is already in the watcher and running in digest cycle do we need to have the apply here?

Reply:
Good point, I will remove it.